### ISSUE TITLE
Config: Allow disabling line-number

### ISSUE THREAD

**Issue by thomas-profitt** [helix-editor/helix#1960]
A previous issue, now closed, suggested "absolute", "relative" and "none" as the valid options for `line-number`,
but only "absolute" and "relative" seem to be implemented; there is no option to disable the line-number column entirely.

**Comment by David-Else** [comment]
+1

I have line numbers turned off 99% of the time in Neovim. In fact I have a toggle in case I need them, in case of some strange emergency:
```
" toggle line numbers
nnoremap <silent><leader>n :set invnumber<CR>
```

**Comment by thomas-profitt** [comment]
Yeah, I like to see my cursor's current line and column number in the status-line (which Helix already shows!), instead of on a side column.

**Comment by David-Kunz** [comment]
Hi,

I've created a [POC PR](https://github.com/helix-editor/helix/pull/1967) to implement this feature request.

Best regards,
David

**Comment by archseer** [comment]
Closed by #1967
